cmd/gpu_plugin/fractional.md

```diff
@@ -0,0 +1,58 @@
+$ kubectl apply -k 'https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/nfd?ref=<RELEASE_VERSION>'
+
+# Create NodeFeatureRules for detecting GPUs on nodes
+$ kubectl apply -k 'https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/nfd/overlays/node-feature-rules?ref=<RELEASE_VERSION>'
+
+# Create GPU plugin daemonset
+$ kubectl apply -k 'https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/gpu_plugin/overlays/fractional_resources?ref=<RELEASE_VERSION>'
+```
+
+### With Device Plugin Operator
+
+Install the Device Plugin Operator according to the [install](../operator/README.md#installation) instructions. When applying the [GPU plugin Custom Resource](../../deployments/operator/samples/deviceplugin_v1_gpudeviceplugin.yaml) (CR), set `resourceManager` option to `true`. The Operator will install all the required RBAC objects and service accounts.
+
+```
+spec:
+  resourceManager: true
+```
+
+## Details about fractional resources
+
+Use of fractional GPU resources requires that the cluster has node extended resources with the name prefix `gpu.intel.com/`. Those are automatically created by GPU plugin with the help of the NFD. When fractional resources are enabled, the plugin lets GAS do card selection decisions based on resource availability and the amount of extended resources requested in the [pod spec](https://github.com/intel/platform-aware-scheduling/blob/master/gpu-aware-scheduling/docs/usage.md#pods).
+
+GAS then annotates the pod objects with unique increasing numeric timestamps in the annotation `gas-ts` and container card selections in `gas-container-cards` annotation. The latter has container separator '`|`' and card separator '`,`'. Example for a pod with two containers and both containers getting two cards: `gas-container-cards:card0,card1|card2,card3`.
+
+Enabling the fractional resource support in the plugin without running GAS in the cluster will only slow down GPU-deployments, so do not enable this feature unnecessarily.
+
+## Tile level access and Level Zero workloads
+
+With multi-tile GPUs, it's possible to assign workloads to different GPU tiles. This support is specific to Level Zero workloads and doesn't apply to other workloads such as OpenCL. When a Pod requests `gpu.intel.com/i915` and `gpu.intel.com/tiles` resources in their pod spec, GAS will select a GPU and its tile (or multiple tiles) for the container. When the GPU plugin receives an allocation request, it adds a Level Zero specific [affinity mask](https://spec.oneapi.io/level-zero/latest/core/PROG.html#affinity-mask) to the container. By default the affinity mask is in the "FLAT" [hierarchy mode](https://spec.oneapi.io/level-zero/latest/core/PROG.html#device-hierarchy). If a container has set the hierarchy environment variable to "COMPOSITE", GPU plugin will adapt the affinity mask accordingly. When tiles are requested for single-tile GPUs, GPU plugin will not set the affinity mask.
```

Comment:
This could also state hierarchy mode variable name explicitly:
```suggestion
With multi-tile GPUs, it's possible to advice workloads to use different GPU tiles. This support is specific to Level Zero workloads and doesn't apply to other workloads such as OpenCL. When a Pod requests `gpu.intel.com/i915` and `gpu.intel.com/tiles` resources in their pod spec, GAS will select a GPU and its tile (or multiple tiles) for the container. When the GPU plugin receives an allocation request, it adds a Level Zero specific [affinity mask](https://spec.oneapi.io/level-zero/latest/core/PROG.html#affinity-mask) to the container. By default the affinity mask is in the "FLAT" [hierarchy mode](https://spec.oneapi.io/level-zero/latest/core/PROG.html#device-hierarchy). If a container spec environment includes hierarchy mode variable with "COMPOSITE" value, GPU plugin will adapt the affinity mask accordingly. When tiles are requested for single-tile GPUs, GPU plugin will not set the affinity mask.
```

Reply:
This is quite a lot of text. Maybe it could also be split into few separate paragraphs for better readability?

In the suggestion:
* I changed  "assign" to "advise" because affinity mask is just an overridable env variable (= won't be respected by all workloads), and
* clarified that hierarchy mode variable needs to be set to container spec, because GPU plugin does not know about it, if it's set e.g. in the container command line (`sh -c "<env>=<mode> <command>`).